src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -78,6 +81,35 @@
           </div>
         </div>
         <br class="hidden-xs">
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="form-group">
+              <h5 class="col-sm-2">
+                <label class="control-label" for="<%= Const.ParamsNames.COURSE_NAME %>">
+                  Course name
+                </label>
+              </h5>
+              <div class="col-sm-10 col-md-8">
+                <c:choose>
+                  <c:when test="${fsForm.courseIdEditable}">
+                    <c:forEach items="${courseAttributes}" var="attributes">
+                      <input class="course-name-data" id="${fn:escapeXml(attributes.id)}"
+                          type="hidden" value="${fn:escapeXml(attributes.name)}">
```

Comment:
Have you checked that this works? According to my slightly outdated jQuery experience, the `.data(...)` can only be called if the attribute is prepended with `data-`, i.e. it should be `data-id` and `data-value`.

Reply:
@wkurniawan07 I recheck your comment and realise I misunderstood what you mean, and I am actually still looping the variable with class ```course-name-data```. With the ```data-*``` method that you suggest, what I could think of is prepend ```data-*``` on the course id and access them by using ```.data(id)```, but one of the problem that I face is that HTML 5 convert their attribute name to lower-case.

For example, course id : CS2014, course name : Computer Science, I will prepend ```data-CS2014 = "Computer Science"``` but what being stored is actually ```cs2014```, but for TEAMMATES course name, we allow course name with same name but different cases to be created, so this causes a problem.

As far as I am concerned, there is no neat way of using ```data-*```, so do you think my current method is fine or I should go further and do more research on how to use ```data-*``` method to achieve our purpose?

Reply:
@huai99 what's wrong with using `data-id="CS2104" data-name="Computer Science"`?
I was never against looping in the first place. Only that `data-*` is a neater solution than having to hard-code `[id][name]` and then parsing it.

Reply:
@wkk91193 Ya, I realise that was a dirty solution, and in my current implementation I have abandoned that, and it is working fine without the ```data-*```, I can access the ```id``` and ```value``` directly

Reply:
@huai99 Ah ok I just checked the JS code. Then it should be fine.